packages/core/src/components/popover/popover.tsx

```diff
@@ -448,6 +460,17 @@ export class Popover extends AbstractPureComponent<IPopoverProps, IPopoverState>
         };
     }
 
+    private getIsOpen(props?: IPopoverProps) {
```

Comment:
arg doesn't seem optional -- no check to handle `undefined`

Reply:
Made it optional because `props` is optional in the constructor. Added a check below.